partner_contact_personal_information_page/README.rst

```diff
@@ -0,0 +1,83 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+===================================
+Contacts' Personal Information Page
+===================================
+
+This module extends the contact management functionality.
+
+Its only purpose is to serve as a base for other modules that add personal
+information fields to **contacts that are persons**.
+
+Ultimately, this module is no longer required in version 9, as each module
+that was previously dependent upon it will declare what it needs for itself.
+
+Installation
+============
+
+This module exists to ensure that upgraded databases continue to work.
+There should be no reason to install it directly, and there should be no
+modules depending on it.
+
+Configuration
+=============
+
+There is no configuration required for this module.
+
+Usage
+=====
+
+You can find a new page called *Personal Information* in the contact's form.
+
+For further information, please visit:
+
+* https://www.odoo.com/forum/help-1
+* https://github.com/OCA/partner-contact/
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/134/11.0
```

Comment:
Minor changes. Cahnge to V12

Reply:
Yes, or use the readme generator to avoid this kind of maintenance in the future.